handbook/company/pricing-features-table.yml

```diff
@@ -478,6 +482,9 @@
       moreInfoUrl: https://github.com/fleetdm/fleet/issues/14920
     - description: Control and restore access to applications by automatically restricting access when devices do not meet particular security requirements.
       moreInfoUrl: https://duo.com/docs/device-health
+  demos:
+    - description: See activity logs when the activities webhook is enabled/disabled and when the destination URL is edited
+      moreInfoUrl: https://youtu.be/3Fndgrsk-Ss 
```

Comment:
@onasismunro and @Drew-P-drawers not sure if this this the right demo to go under "Device health"

I think we want a demo that focuses on the goal of notifying or blocking misconfigured devices (see top level `description`).

If we can't find one quickly, I think let's remove this demo from this PR so we can merge in the other changes.

Reply:
I think it's already under audit logging, which makes sense. Removing this! thanks for catching this.